components/camel-casper/src/main/java/org/apache/camel/component/casper/CasperConsumer.java

```diff
@@ -0,0 +1,78 @@
+package org.apache.camel.component.casper;
+
+import java.net.URI;
+import java.time.Duration;
+import java.util.concurrent.CountDownLatch;
+
+import org.apache.camel.Processor;
+import org.apache.camel.ShutdownRunningTask;
+import org.apache.camel.Suspendable;
+import org.apache.camel.spi.ShutdownAware;
+import org.apache.camel.support.DefaultConsumer;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.launchdarkly.eventsource.EventHandler;
+import com.launchdarkly.eventsource.EventSource;
+
+/**
+ * Camel CasperComsumer Component
+ * 
+ * @author mabahma
+ *
+ */
+
+public class CasperConsumer extends DefaultConsumer implements ShutdownAware, Suspendable {
+	public static final Logger logger = LoggerFactory.getLogger(CasperConsumer.class);
```

Comment:
Checkstyle will catch this, but as a general comment: the `logger` variable needs to comply with the naming standards defined in our checkstyle rules. As such, I suggest naming it as `LOG`.

Reply:
fixed